tests/installation/first_boot.pm

```diff
@@ -20,7 +20,7 @@ use testapi;
 sub handle_login {
     mouse_hide();
     wait_still_screen;
-    if (get_var('DM_NEEDS_USERNAME')) {
+    if (get_var('DM_NEEDS_USERNAME') || check_screen 'displaymanager-username-prompt', 3) {
```

Comment:
would `check_screen '…', 0` also work here? There is a `wait_still_screen` in before so there should not be any change